### PR TITLE
Update eBPF documentation to reflect default enabled status

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

The main motivation behind the PR is to follow up on #8665, which enables eBPF by default on new installations. The documentation now reflects that while also directing users with older/non-nightly versions to explicitly enable it. In addition, I moved some of the old information about KPROBES and the required filesystems to a **troubleshooting** section.

I also took this opportunity to run through the doc and make additional fixes to the text where applicable.

##### Component Name

collectors

### CODE REVIEW

File: collectors/ebpf.plugin/README.md [hunk 1]
```diff
@@ -1,141 +1,67 @@
...
-
--   [Ubuntu](https://wiki.ubuntu.com/Kernel/BuildYourOwnKernel)
--   [Debian](https://kernel-team.pages.debian.net/kernel-handbook/ch-common-tasks.html#s-common-official)
--   [Fedora](https://fedoraproject.org/wiki/Building_a_custom_kernel)
--   [CentOS](https://wiki.centos.org/HowTos/Custom_Kernel)
--   [Arch Linux](https://wiki.archlinux.org/index.php/Kernel/Traditional_compilation)
--   [Slackware](https://docs.slackware.com/howtos:slackware_admin:kernelbuilding)
+**The eBPF collector is installed and enabled by default on new nightly installations of the Agent**. eBPF monitoring
+only works on Linux systems and with specific Linux kernels, including all kernels newer than `4.11.0`, and all kernels
+on CentOS 7.6 or later._
```
Comment (prologic) [inline collectors/ebpf.plugin/README.md:31]:
```suggestion
on CentOS 7.6 or later.
```

File: collectors/ebpf.plugin/README.md [hunk 2]
```diff
@@ -178,35 +104,129 @@ process and thread creation.
...
+-   `return`: In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
+    new charts for the return of these functions, such as errors. Monitoring function returns can help in debugging
+    software, such as failing to close file descriptors or creating zombie processes.
+
+## Install and enable the eBPF collector
+
+Systems running _v1.22 or earlier_ of the Agent will need to install and enable the eBPF collector manually. The
+collector is enabled by default on later versions of the Agent.
+
+If you installed via the one-line installation script, 64-bit binary, or manually, you can append the `--enable-ebpf`
```
Comment (thiagoftsm) [inline collectors/ebpf.plugin/README.md:145]: the `--enable-ebpf` has been removed, please, remove the reference to it.

File: collectors/ebpf.plugin/README.md [hunk 3]
```diff
@@ -178,35 +104,129 @@ process and thread creation.
...
+option when you reinstall.
+
+For example, if you used the [one-line installation
+script](/packaging/installer/README.md#automatic-one-line-installation-script), you can reinstall Netdata with the
+following:
+
+```bash
+bash <(curl -Ss https://my-netdata.io/kickstart.sh) --enable-ebpf
```
Comment (thiagoftsm) [inline collectors/ebpf.plugin/README.md:153]: Please, also remove the option `--enable-ebpf` here

**Comment by joelhans** [comment]
@prologic If `check-kernel-config.sh` returns no output that means the system is ready for eBPF, correct? Happy to add it in, since I didn't even know it existed, but I want to make sure I'm writing the correct thing.

File: collectors/ebpf.plugin/README.md (hunk 2 above)
Comment (joelhans) [reply collectors/ebpf.plugin/README.md]: Done!

**Comment by prologic** [comment]
> @prologic If `check-kernel-config.sh` returns no output that means the system is ready for eBPF, correct? Happy to add it in, since I didn't even know it existed, but I want to make sure I'm writing the correct thing.

Correct.

**Comment by Ferroin** [comment]
Travis build is actually passed, it just didn't report back to GitHub (again). Merging this as it's otherwise ready.

**Pull request merged**